src/components/studies/scheduler/Duration.tsx

```diff
@@ -104,6 +106,15 @@ const Duration: React.FunctionComponent<
         {...props}
         id={numberLabel.replace(' ', '')}
         onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
+          if (inputDurationCapInWeeks !== undefined) {
```

Comment:
These checks should be a in a separate function which is called from the existing `changeValue` fn